modules/runtime/nuxeo-launcher-commons/src/main/java/org/nuxeo/launcher/config/ServerConfigurator.java

```diff
@@ -92,23 +94,37 @@
     /** @since 9.3 */
     public static final String JAVA_OPTS = "JAVA_OPTS";
 
-    private static final String NEW_FILES = ConfigurationGenerator.TEMPLATES + File.separator + "files.list";
+    private static final String NEW_FILES = "files.list";
 
     protected final ConfigurationGenerator generator;
 
+    /** @since 11.5 */
+    protected final ConfigurationHolder configHolder;
+
+    @Deprecated
     protected File dataDir = null;
```

Comment:
Could you remove all the useless `= null`?

Reply:
The whole class is deleted in the end, I will just make sure I didn't add intermediate changes.